<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="user" name="com.hrms.Ex_ExcelDataRetrieve" tests="2" failures="1" timestamp="8 Feb 2019 08:20:30 GMT" time="7.157" errors="0">
  <testcase name="pim" time="0.000" classname="com.hrms.Ex_ExcelDataRetrieve">
    <failure type="java.lang.RuntimeException" message="java.io.FileNotFoundException: ?E:\Book1.xls (The filename, directory name, or volume label syntax is incorrect)">
      <![CDATA[java.lang.RuntimeException: java.io.FileNotFoundException: ?E:\Book1.xls (The filename, directory name, or volume label syntax is incorrect)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:143)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:426)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1383)
	at org.testng.internal.Invoker.createParameters(Invoker.java:1075)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1180)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
	at org.testng.TestNG.run(TestNG.java:1031)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.io.FileNotFoundException: ?E:\Book1.xls (The filename, directory name, or volume label syntax is incorrect)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(Unknown Source)
	at java.io.FileInputStream.<init>(Unknown Source)
	at jxl.Workbook.getWorkbook(Workbook.java:213)
	at jxl.Workbook.getWorkbook(Workbook.java:198)
	at com.hrms.Ex_ExcelDataRetrieve.AdminServantMaidList(Ex_ExcelDataRetrieve.java:68)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:117)
	... 20 more
]]>
    </failure>
  </testcase> <!-- pim -->
  <testcase name="login" time="7.157" classname="com.hrms.Ex_ExcelDataRetrieve"/>
</testsuite> <!-- com.hrms.Ex_ExcelDataRetrieve -->
